lib/cn/hblock_reader.h

```diff
@@ -18,6 +18,7 @@ struct mblock_props;
 struct mpool;
 struct mpool_mcache_map;
 struct wbt_desc;
+struct vgmap;
```

Comment:
I think naming the struct as `vgroup_map` would be better. vgmap is a good name for variables.

Reply:
We finalized this name in one of the earlier comments - vgmap for the struct name and vgmap_*() for the functions.